sdk/resources/Microsoft.Azure.Management.Resource/src/Microsoft.Azure.Management.ResourceManager.csproj

```diff
@@ -10,7 +10,7 @@
 			Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Resources" (https://www.nuget.org/packages/Azure.ResourceManager.Resources/). Please see the package changelog for more information.
 		</Description>
 		<AssemblyName>Microsoft.Azure.Management.ResourceManager</AssemblyName>
-		<Version>4.0.0</Version>
+		<Version>3.8.0-preview</Version>
```

Comment:
Previous was 3.7.4-preview, this should be 3.7.5-preview

Reply:
I bumped it up since we are using a new swagger Resources API version with this build